pjsip/src/pjsip-ua/sip_siprec.c

```diff
@@ -0,0 +1,276 @@
+#include <pjsip-ua/sip_siprec.h>
```

Comment:
Put copyright and licensing info at the top of the file, also in the header file.

You can put your name as an author (optional, if you wish so).